contracts/RelayHub.sol

```diff
@@ -70,6 +70,7 @@ contract RelayHub is RelayHubApi {
      * Unused deposited can be withdrawn with `withdraw()`
      */
     function depositFor(address target) public payable {
+        require(msg.value <= minimum_stake, "deposit too big");
```

Comment:
remove requirement: deposit can't depend on stake

Reply:
We remove it in a separate commit